test_tools/sigma_verify_rules.py

```diff
@@ -21,80 +21,20 @@
 """
 
 
+
 import logging
 import os
 import argparse
 import sys
-import codecs
-import sigma.parser.exceptions
-import sigma.configuration as sigma_configuration
+# TODO: not sure if there is a better way
+sys.path.append('../timesketch/lib/')
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+import sigma_util as ts_sigma_lib
```

Comment:
This causes a linter issue as it is not possible to import, I do not know how to fix it in a different way.

Reply:
why not just `from timesketch.lib import sigma_util` ?

Reply:
because it is in test_tools. So timesketch.lib would look into ```test_tools/timesketch/lib```or am I wrong?

Reply:
the file is stored in test_tools/timesketch/lib and the way you run the tool is to run it within the test_Tool folder where you run it using `PYTHONPATH=. python ....` meaning that that is the only way to run this, otherwise this will fail.

and yes, it is stored there and thereore should be imported from there